### PR TITLE
Support plaintext responses

### PR DESCRIPTION
**Pull request by SasSwart**
This pull request adds support for text/plain content types in go-gin generated servers.
It also moves some of the templating in the go-gin server templates to more appropriate locations to enhance clarity and readability.

### CODE REVIEW

File: render/engine.go [hunk 1]
```diff
@@ -48,14 +48,14 @@ func (e Engine) BuildRenderNode() tree.TraversalFunc {
 			templateFile = "openapi.tmpl"
 		case *path.Item:
 			templateFile = "path_item.tmpl"
```
Comment (SasSwart) [inline render/engine.go:50]: This sorts these hierarchicaly

File: templates/go-gin/operation.tmpl [hunk 2]
```diff
@@ -3,27 +3,24 @@
...
-{{- if ne $property.MinLength 0 }}{{ $needs_fmt = true}}{{- end }}
-{{- if ne $property.MaxLength 0 }}{{ $needs_fmt = true}}{{- end }}
 {{- if ne $property.Pattern "" }}{{ $needs_regexp = true }}{{- end }}
 {{- end }}
 {{- end }}
 
-{{ if $needs_fmt -}} import "fmt" {{- end }}
 {{ if $needs_regexp -}} import "regexp" {{- end }}
 
+var _ = fmt.Sprint("")
```
Comment (alexthemayers) [inline templates/go-gin/operation.tmpl:22]: Why is this here?

Comment (SasSwart) [reply templates/go-gin/operation.tmpl:22]: So that I can remove all the `$needs_fmt` stuff.

Comment (SasSwart) [reply templates/go-gin/operation.tmpl:22]: Essentially I'm making it always need fmt.

**Review by alexthemayers — APPROVED** [review]
